replication_stats.go

```diff
@@ -3,17 +3,56 @@ package sgreplicate
 import "sync"
 
 type ReplicationStats struct {
-	lock             sync.Mutex
-	docsRead         uint32
-	docsWritten      uint32
-	docWriteFailures uint32
-	startLastSeq     uint32
-	endLastSeq       interface{}
+	lock                       sync.RWMutex
+	docsRead                   uint32
+	docsWritten                uint32
+	docWriteFailures           uint32
+	startLastSeq               uint32
+	numAttachmentsTransferred  uint32
+	attachmentBytesTransferred uint32
+	docsCheckedSent            uint32
```

Comment:
Any reason all these shouldn't be uint64?  seq is usually a uint64, byte count seems like a potential candidate for large values.

Reply:
Agree.  Fixed.